src/Snak/SnakList.php

```diff
@@ -211,9 +208,6 @@ private function getNewOffset() {
 	 */
 	public function offsetUnset( $index ) {
 		if ( $this->offsetExists( $index ) ) {
-			/**
-			 * @var Hashable $element
```

Comment:
I assume this could also be `@var Snak $element`? Or can the `$element` also be something else?

Reply:
I just didn't see the reason to keep it as there was no consistent rule regarding where an interface or class of a variable is mentioned, don't mind putting it back though if it's a blocker.

Reply:
I think the rule is more or less to include `@var` comments wherever “the IDE” can’t figure out the type on its own; in this case, `offsetGet` is a general `ArrayAccess` method, so there’s no way for the IDE to know that `$element` will be anything other than `mixed`. (At least, this is still the case in IntelliJ 202.7502.)